plugins/parallel-brushing.js

```diff
@@ -0,0 +1,140 @@
+(function (factory) {
+    if (typeof define === 'function' && define.amd) {
+        define(['tauCharts'], function (tauPlugins) {
+            return factory(tauPlugins);
+        });
+    } else if (typeof module === 'object' && module.exports) {
+        var tauPlugins = require('tauCharts');
+        module.exports = factory(tauPlugins);
+    } else {
+        factory(this.tauCharts);
+    }
+})(function (tauCharts) {
+
+    var _ = tauCharts.api._;
+
+    function ChartParallelBrushing(xSettings) {
+
+        var settings = _.defaults(
+            xSettings || {},
+            {
+                verbose: false,
+                forceBrush: {}
+            });
+
+        var plugin = {
+
+            init: function (chart) {
+                if (settings.verbose) {
+                    plugin.panel = chart.insertToRightSidebar(this.template());
+                }
+
+                chart.traverseSpec(
+                    chart.getConfig(),
+                    function (unit) {
+                        if (unit && unit.type === 'COORDS.PARALLEL') {
```

Comment:
i think, we should move this condition in tauChart api like https://github.com/TargetProcess/tauCharts/blob/5170fe0e2cad1ea9f37581eb26c191cdec6816f6/src/tau.charts.js#L46